app/web/template/choose_replacement_attorneys.gohtml

```diff
@@ -3,7 +3,37 @@
 {{ define "main" }}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-           Not done yet
+            <form novalidate method="post">
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                            <h1 class="govuk-fieldset__heading">{{ tr .App "whoWouldYouLikeToBeTheReplacementAttorneys" }}</h1>
+                        </legend>
+
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">{{ tr .App "replacementAttorney" }}</legend>
```

Comment:
<img width="713" alt="image" src="https://user-images.githubusercontent.com/146390/201327084-7815bac6-096a-429e-ae88-f3a0fcccfeb7.png">
I think the double header looks weird

Reply:
Agree - as there was no designs to work from I'm waiting on the whole flow to be done for a mass tweak of the content and design rather than trying to come up with something myself.